src/api/translation-api.ts

```diff
@@ -1,28 +1,46 @@
 import { debug, setFailed } from '@actions/core';
+import Axios, { AxiosRequestConfig } from 'axios';
+import { v4 } from 'uuid';
 import { AvailableTranslations } from '../abstractions/available-translations';
 import { TranslationResult, TranslationResults, TranslationResultSet } from '../abstractions/translation-results';
-import { v4 } from 'uuid';
-import Axios, { AxiosRequestConfig } from 'axios';
 import { TranslatorResource } from '../abstractions/translator-resource';
-import { chunk } from '../helpers/utils';
 import { toResultSet } from '../helpers/api-result-set-mapper';
+import { batch, chunk } from '../helpers/utils';
 
 /**
 * https://docs.microsoft.com/azure/cognitive-services/translator/language-support#translate
-*/ 
+*/
 export const getAvailableTranslations = async (): Promise<AvailableTranslations> => {
     const apiUrl = 'https://api.cognitive.microsofttranslator.com/languages';
     const query = 'api-version=3.0&scope=translation';
     const response = await Axios.get<AvailableTranslations>(`${apiUrl}?${query}`);
 
     return response.data;
-}
+};
 
 export const translate = async (
     translatorResource: TranslatorResource,
     toLocales: string[],
-    translatableText: Map<string, string>): Promise<TranslationResultSet | undefined> => {
+    translatableText: Map<string, string>,
+    filePath: string): Promise<TranslationResultSet | undefined> => {
```

Comment:
What is the `filePath` being used for, it is hard to tell?

Reply:
It used only to improve error messages. Previously it was impossible to know which file causes a problem.